src/apispec/ext/marshmallow/__init__.py

```diff
@@ -91,7 +92,7 @@ def init_spec(self, spec):
         super().init_spec(spec)
         self.spec = spec
         self.openapi_version = spec.openapi_version
-        self.openapi = OpenAPIConverter(
+        self.openapi = self.Converter(
```

Comment:
I meant the instance attribute:

```suggestion
        self.converter = self.Converter(
```

Reply:
Oh, sure...

Reply:
I don't care either way.

I pushed a new commit with this change.

Reply:
Looks good. Go ahead and merge if you’re happy with this